src/hexkit/providers/mongokafka/provider.py

```diff
@@ -396,6 +398,16 @@ async def republish(self) -> None:
 class MongoKafkaConfig(MongoDbConfig, KafkaConfig):
     """Config parameters and their defaults."""
 
+    @field_validator("kafka_max_message_size", mode="after")
+    @classmethod
+    def validate_max_message_size(cls, value: int) -> int:
+        """Validate the maximum message size."""
+        if value > 16_000_000:
```

Comment:
Seems to be 16 MB, not 16 MiB according to [docs](https://www.mongodb.com/docs/v5.2/reference/limits/#mongodb-limit-Sharding-Existing-Collection-Data-Size)

Reply:
Thanks for spotting that